lib/aggregators/log-aggregator.js

```diff
@@ -27,6 +27,19 @@ class LogAggregator extends EventAggregator {
   }
 
   _toPayloadSync() {
+    if (['starting', 'stopped'].includes(this.agent._state)) {
+      logger.debug('Agent not ready. Not harvesting logs...')
+      return
+    } else if (
+      this.agent.config &&
+      this.agent.config.event_harvest_config &&
+      this.agent.config.event_harvest_config.harvest_limits &&
+      this.agent.config.event_harvest_config.harvest_limits.log_event_data === 0
+    ) {
+      logger.debug('Log aggregation is disabled.')
```

Comment:
You might want to call this out as `APM logs are disabled for your account.` or something to that effect as the big red button page header is `APM log settings`